modules/apps/headless/headless-builder/headless-builder-test/src/testFunctional/tests/[UI]DefineAPISchemasWithinAnAPIGroup/CreateAPIApplicationSchema.testcase

```diff
@@ -93,9 +93,9 @@ definition {
 			Click(locator1 = "APIBuilder#SELECT_AN_OBJECT_DEFINITION");
 		}
 
-		task ("Then all object deinifition entities from Objects are listed inluding the one placed on 2nd page of Objects") {
+		task ("Then all modifiable object definition entities from Objects are listed including the one placed on 2nd page of Objects") {
 			AssertElementPresent(
-				index = 21,
+				index = 13,
```

Comment:
Why is this 13 now? Is this something that could change again in the future?

Reply:
This is because of this new [implementation](https://github.com/brianchandotcom/liferay-portal/pull/149990). Before the PR, we took into account all Object Definitions that can be modifiable or not. So the final result of this just to indicate all that are modifiable